.circleci/config.yml

```diff
@@ -2,8 +2,13 @@ version: 2.1
 orbs:
   coveralls: coveralls/coveralls@1.0.6
   node: circleci/node@1.1.6
+workflows:
```

Comment:
can you have this in a different PR?

Reply:
Want to separate the oTokenFactory changes from the circle changes

Reply:
Yeah Haythem added this just to test the code coverage. Let me drop these commits.